<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REDTabOverview</class>
 <widget class="QWidget" name="REDTabOverview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>REDTabOverview</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="gbox_running_processes">
     <property name="title">
      <string>Running Processes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="glayout_running_processes">
        <item row="0" column="0">
         <widget class="QListView" name="lview_running_processes"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pbutton_refresh">
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="gbox_status">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="glayout_status">
        <item row="0" column="1">
         <widget class="QProgressBar" name="pbar_cpu">
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>Please wait...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="pbar_memory">
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>Please wait...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_cpu">
          <property name="text">
           <string>CPU</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QProgressBar" name="pbar_storage">
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>Please wait...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_memory">
          <property name="text">
           <string>Memory</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_storage">
          <property name="text">
           <string>Storage</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
